website/docs/enterprise/get-started.md

```diff
@@ -2,20 +2,20 @@
 title: Get started
 ---
 
-Installing authentik is exactly the same process for both Enterprise version and our free [open source](https://github.com/goauthentik/authentik) version.
+Installing authentik is exactly the same process for both Enterprise version and our open source version.
```

Comment:
Not sure we want to remove the word "free". I guess open source does indeed mean free, but doesn't hurt to be very clear.